### ISSUE TITLE
Υποδομή συστήματος

### ISSUE THREAD

**Issue by Jimmys20** [Jimmys20/ptyxiaki#1]
Πρέπει να καθοριστεί ποια θα είναι η υποδομή του συστήματος. Προτείνω να στηθεί το project σε ένα virtual machine με

- **λειτουργικό σύστημα** [CentOS](https://www.centos.org/),
- **βάση δεδομένων** [Oracle XE](https://www.oracle.com/database/technologies/appdev/xe.html) και
- [Apache](https://httpd.apache.org/) **web server**

Η Oracle XE (Express Edition) δίνει storage μέχρι 12GB. Αν υπάρχει περίπτωση να μην φτάσουν (αν και δεν το νομίζω) τότε θα πρέπει να χρησιμοποιηθεί κάποια άλλη βάση δεδομένων.

Η εφαρμογή θα γίνει με χρήση

- [ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/?view=aspnetcore-2.1) και 
- [Angular](https://angular.io/)

Εφόσον οριστικοποιηθούν αυτά μπορεί να πλέον να ξεκινήσει το development.

**Comment by dima1270** [comment]
Περιμένω τον Κωστίκα Βισνία να γυρίσει με άδεια (είναι στο στρατό) κατά τις αρχές Δεκεμβρίου. Πιστεύω ότι επειδή αυτός έστεισε τα VM στο τμήμα θα μας δώσει ποιο γρήγορα και σαφέστερες απαντήσεις από τον κ. Ψαρρά. άλλωστε αυτός το προτεινες γιατί δεν γνωρίζει πως έχει στείσει το σύστημα με τα VM o Βισνία.
Τα στοιχεία που ζήτησες θα σου τα δώσω ως απάντηση στο e-mail όχι από δω με ανοιχτή συζήτηση

**Comment by Jimmys20** [comment]
Ναι, οι "ευαίσθητες" πληροφορίες θα πρέπει να ανταλλάσονται με κάποιον άλλο τρόπο και όχι εδώ. Γι' αυτό έγραψα στο e-mail ότι το repository είναι δημόσιο και στη συζήτηση εδώ δεν ανέφερα κάτι για τα άλλα στοχεία που ζήτησα. Θα μεταφέρω τις απορίες μου στον Κώστα ώστε όταν έχει ελεύθερο χρόνο να βοηθήσει.
